code/sound/sound.cpp

```diff
@@ -1145,6 +1187,38 @@ int snd_is_playing(sound_handle sig)
 	return 0;
 }
 
+// ---------------------------------------------------------------------------------------
+// snd_is_paused()
+//
+// Determine if a sound is paused
+//
+// returns:			1				=> sound is currently paused
+//						0				=> sound is not paused
+//
+// parameters:		sig	=> signature of sound, what is returned from snd_play()
+//
+int snd_is_paused(sound_handle sig)
+{
+	int channel, is_paused;
```

Comment:
I know other reviewers say to "match the formatting of the surrounding code", but imho, variables should be declared as close to their assignment as possible, even if the surrounding code is not doing that.  I won't insist on this change, but it would fewer lines.

Reply:
and Done